azurecontainerapps.ts

```diff
@@ -653,16 +646,14 @@ export class azurecontainerapps {
             return;
         }
 
-        if (this.noIngressUpdate && !this.shouldCreateOrUpdateContainerAppWithUp) {
+        if (this.noIngressUpdate) {
             // Update the Container Registry details on the existing Container App, if provided as an input
             if (!this.util.isNullOrEmpty(this.registryUrl) && !this.util.isNullOrEmpty(this.registryUsername) && !this.util.isNullOrEmpty(this.registryPassword)) {
                 await this.appHelper.updateContainerAppRegistryDetails(this.containerAppName, this.resourceGroup, this.registryUrl, this.registryUsername, this.registryPassword);
             }
 
             // Update the Container App using the 'update' command
             await this.appHelper.updateContainerApp(this.containerAppName, this.resourceGroup, this.commandLineArgs);
-        } else if (this.shouldCreateOrUpdateContainerAppWithUp) {
-            await this.appHelper.createOrUpdateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs);
         } else if (this.adminCredentialsProvided && !this.noIngressUpdate) {
             // Update the Container App with `up` command when admin credentials are provided and ingress is manually provided.
             await this.appHelper.updateContainerAppWithUp(this.containerAppName, this.resourceGroup, this.commandLineArgs, this.ingress, this.targetPort);
```

Comment:
we still appear to be updating with the Up command here

Reply:
Yes, we still need these. This change just updates container app for only source to cloud scenarios. There are other scenarios that still require up and is already being handled